doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -259,8 +263,7 @@ pass testing for ``start_test -performance``).
 
 .. _Creating a Simple Test: https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/TestSystem.rst#creating-a-simple-test
```

Comment:
This link doesn't seem to directly lead somewhere in the associated doc (and should probably point to the Sphinx page anyways), I think the heading got renamed or something